drake/examples/Pendulum/test/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+include_directories(${PROJECT_SOURCE_DIR}/core)
+include_directories(${PROJECT_SOURCE_DIR}/systems)
+include_directories(${PROJECT_SOURCE_DIR}/systems/frames)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants)
+include_directories(${PROJECT_SOURCE_DIR}/systems/plants/joints)
+include_directories(${PROJECT_SOURCE_DIR}/systems/controllers)
+include_directories(${PROJECT_SOURCE_DIR}/util)
+
+if (LCM_FOUND AND snopt_c_FOUND)
+  add_executable(pendulumURDFDynamicsTest urdfDynamicsTest.cpp)
```

Comment:
Why does this need snopt? Also shouldn't need LCM to test the dynamics of a pendulum.